src/Compilers/CSharp/Portable/Parser/LanguageParser_InterpolatedString.cs

```diff
@@ -130,32 +126,48 @@ SyntaxToken getCloseQuote(Range openQuoteRange)
                 // Make a token for the close quote " (even if it was missing)
                 var closeQuoteText = originalText[closeQuoteRange];
                 return closeQuoteText == ""
-                    ? SyntaxFactory.MissingToken(SyntaxKind.InterpolatedStringEndToken).TokenWithTrailingTrivia(originalToken.GetTrailingTrivia())
-                    : SyntaxFactory.Token(null, SyntaxKind.InterpolatedStringEndToken, closeQuoteText, closeQuoteText, originalToken.GetTrailingTrivia());
+                    ? SyntaxFactory.MissingToken(leading: null, SyntaxKind.InterpolatedStringEndToken, trailing: originalToken.GetTrailingTrivia())
+                    : SyntaxFactory.Token(leading: null, SyntaxKind.InterpolatedStringEndToken, closeQuoteText, closeQuoteText, originalToken.GetTrailingTrivia());
             }
         }
 
-        private static InterpolationSyntax ParseInterpolation(CSharpParseOptions options, string text, Lexer.Interpolation interpolation, bool isVerbatim)
+        private static InterpolationSyntax ParseInterpolation(
+            CSharpParseOptions options,
+            string text,
+            Lexer.Interpolation interpolation,
+            Lexer.InterpolatedStringKind kind)
         {
-            // Grab from before the { all the way to the start of the } (or the start of the : if present).  The parsing
-            // of the colon and/or close curly is specially handled in ParseInterpolation below.
-            var parsedText = text[new Range(
-                interpolation.OpenBraceRange.Start,
+            // Grab the text from after the { all the way to the start of the } (or the start of the : if present). This
+            // will be used to parse out the expression of the interpolation.
+            //
+            // The parsing of the open brace, close brace and colon is specially handled in ParseInterpolation below.
+            var expressionText = text[new Range(
+                interpolation.OpenBraceRange.End,
                 interpolation.HasColon ? interpolation.ColonRange.Start : interpolation.CloseBraceRange.Start)];
 
-            // TODO: some of the trivia in the interpolation maybe should be trailing trivia of the openBraceToken
```

Comment:
this TODO is what thsi PR fixes.  the trivia after the `{` belongs to it, not the expression we parse between `{` and `}`